gallery/src/GalleryNav/GalleryNavIcons.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+export const GalleryNavIconCSS3 = () => (
```

Comment:
Surely we need to provide attribution for this one manually, since we copied it into our code manually.

Reply:
> Surely we need to provide attribution for this one manually, since we copied it into our code manually.

@entroform created the icon from scratch, we don't need to attribute it, it's ours.

Reply:
> @entroform created the icon from scratch, we don't need to attribute it, it's ours.

My original proposition, which I believe he followed, was to pull those CSS block letters from the CC-licensed icons seen here: https://commons.wikimedia.org/wiki/File:CSS3_and_HTML5_logos_and_wordmarks.svg

Reply:
> > @entroform created the icon from scratch, we don't need to attribute it, it's ours.
> 
> My original proposition, which I believe he followed, was to pull those CSS block letters from the CC-licensed icons seen here: https://commons.wikimedia.org/wiki/File:CSS3_and_HTML5_logos_and_wordmarks.svg

He ended up redoing it because when we pulled the letters out they were too small. So @entroform created a brand new file that looked better at the smaller size